From eb7c1e7fb16338d8fc03a0bb0f9bb53eb99ee139 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 7 May 2016 00:43:33 -0700
Subject: [PATCH] XQuartz: Fix default CFBundleVersion

m4/shell variable name collision broke the case when the configure option was not used

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 configure.ac | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2633f4e..cf13123 100644
--- a/configure.ac
+++ b/configure.ac
@@ -559,11 +559,10 @@ AC_ARG_WITH(bundle-id-prefix,  AS_HELP_STRING([--with-bundle-id-prefix=RDNS_PREF
                                [ BUNDLE_ID_PREFIX="${withval}" ])
 AC_SUBST([BUNDLE_ID_PREFIX])
 AC_DEFINE_UNQUOTED(BUNDLE_ID_PREFIX, "$BUNDLE_ID_PREFIX", [Prefix to use for bundle identifiers])
-DEFAULT_BUNDLE_VERSION=`echo ${PACKAGE_VERSION} | cut -f1-3 -d.`
-m4_define(DEFAULT_BUNDLE_VERSION, m4_esyscmd([echo ]AC_PACKAGE_VERSION[ | cut -f1-3 -d.]))
+m4_define(DEFAULT_BUNDLE_VERSION, m4_esyscmd([echo ]AC_PACKAGE_VERSION[ | cut -f1-3 -d. | tr -d '\n']))
 AC_ARG_WITH(bundle-version,    AS_HELP_STRING([--with-bundle-version=VERSION], [Version to use for X11.app's CFBundleVersion (default: ]DEFAULT_BUNDLE_VERSION[)]),
                                [ BUNDLE_VERSION="${withval}" ],
-                               [ BUNDLE_VERSION="${DEFAULT_BUNDLE_VERSION}" ])
+                               [ BUNDLE_VERSION="DEFAULT_BUNDLE_VERSION" ])
 AC_SUBST([BUNDLE_VERSION])
 AC_ARG_WITH(bundle-version-string, AS_HELP_STRING([--with-bundle-version-string=VERSION], [Version to use for X11.app's CFBundleShortVersionString (default: ]AC_PACKAGE_VERSION[)]),
                                [ BUNDLE_VERSION_STRING="${withval}" ],
-- 
2.8.2

